AIPaddle.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <SFML/Graphics.hpp>
+#include "Paddle.hpp"
+
+class AIPaddle : public Paddle {
+public:
+  // Inheret Paddle's constructor
+  using Paddle::Paddle;
+  
+  void update(sf::RenderWindow& window) override {
+    //only move if paddle will stay on screen
```

Comment:
it would be more intuitive if the AI paddle had the up and down arrow keys and the player had the W and S keys.